airbyte-integrations/connectors/source-clazar/integration_tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+#
```

Comment:
If there are no actual integration tests, would you please remove the python files in this directory?